lib/svgo/coa.js

```diff
@@ -295,9 +304,11 @@ function optimizeFromString(svgstr, config, datauri, input, output) {
                 output = input;
             }
 
-            console.log('\r');
+            if (! config.quiet) {
```

Comment:
Thanks, I have updated my pull request with a fix. Is there any way to check for style inconsistencies with this project? I found a jshint configuration file and ran jshint, but it did not report anything.